src/stable/build/commands/compile/wasm_binary/rust/stable_canister_template/src/candid.rs

```diff
@@ -34,21 +34,36 @@ fn initialize_and_get_candid() -> Result<CCharPtr, Box<dyn Error>> {
     quickjs_with_ctx(|ctx| -> Result<CCharPtr, Box<dyn Error>> {
         let globals = ctx.globals();
 
-        globals.set("_azleNodeWasmEnvironment", true)?;
+        globals.set("_azleActions", Array::new(ctx.clone()))?;
 
-        globals.set("exports", Object::new(ctx.clone())?)?;
+        globals.set("_azleCanisterMethodNames", Object::new(ctx.clone())?)?;
 
         globals.set("_azleExperimental", false)?;
 
-        globals.set("_azleCanisterMethodNames", Object::new(ctx.clone())?)?;
+        globals.set("_azleExportedCanisterClassInstance", Undefined)?;
```

Comment:
Some of these are new, but some of the look familiar to me. Are we just explicitly setting them here?

Reply:
My current guess is that if they have never been set it causes problems with the state machine?